estate/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    'name': 'Estate',
+    'author': 'mobar',
```

Comment:
```suggestion
    'author': 'Odoo S.A.',
```
As an employee, you work in the name of Odoo.